controllers/api/v1alpha1/runworkload_types.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// RunWorkloadSpec defines the desired state of RunWorkload
+type RunWorkloadSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// +kubebuilder:validation:Required
+	GUID             string                        `json:"GUID"`
+	Version          string                        `json:"version"`
+	AppGUID          string                        `json:"appGUID"`
+	ProcessType      string                        `json:"processType"`
+	Image            string                        `json:"image"`
+	ImagePullSecrets []corev1.LocalObjectReference `json:"imagePullSecrets"`
+	Command          []string                      `json:"command,omitempty"`
+	Env              []corev1.EnvVar               `json:"env,omitempty"`
+	Health           Healthcheck                   `json:"health"`
+	Ports            []int32                       `json:"ports,omitempty"`
+	// +kubebuilder:default:=1
+	Instances int32 `json:"instances"`
+	MemoryMB  int64 `json:"memoryMB"`
+	// +kubebuilder:validation:Minimum:=1
+	// +kubebuilder:validation:Required
+	DiskMB int64 `json:"diskMB"`
```

Comment:
Maybe we rename these to `MiB` instead of `MB`? See https://github.com/cloudfoundry/korifi/issues/1233#issuecomment-1162039238

Reply:
done